feat(ui): make git repository dropdown searchable

favourch opened the pull request:
## Changes

- feat(ui): replace `x-forms.select` with `x-forms.datalist` for the repository picker in `resources/views/livewire/project/new/github-private-repository.blade.php` to make dropdown searchable

favourch committed:
feat(project/new): make GitHub repository selection searchable

- Replace x-forms.select with x-forms.datalist for repository picker
- Enable type-to-filter UX with Alpine.js, keeping Livewire binding
- Update in resources/views/livewire/project/new/github-private-repository.blade.php

favourch committed:
Merge branch 'v4.x' into feat/github-app-repo-search-dropdown

ShadowArcanist reviewed (changes requested):
Please change the target branch to NEXT instead of `v4.x` and did you manually test your changes?

favourch commented:
Alright. I’ll change the target branch to next. I also tested my changes.

On Fri, Oct 31, 2025 at 5:49 AM ShadowArcanist ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Please change the target branch to NEXT instead of v4.x and did you
> manually test your changes?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/coollabsio/coolify/pull/7064#pullrequestreview-3404131499>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFRIUILJ22PW6Q555KPHMD32NLFFAVCNFSM6AAAAACKYPGNLOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTIMBUGEZTCNBZHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

favourch reviewed (commented):
@ShadowArcanist I've updated the target branch from `v4.x` to `next` as requested. 

The changes remain the same:
- Replaced `x-forms.select` with `x-forms.datalist` for searchable repository selection
- All functionality preserved with improved UX

Ready for review!

ShadowArcanist commented:
@coderabbit review

ShadowArcanist commented:
> @ShadowArcanist I've updated the target branch from `v4.x` to `next` as requested.
> 
> The changes remain the same:
> 
>     * Replaced `x-forms.select` with `x-forms.datalist` for searchable repository selection
> 
>     * All functionality preserved with improved UX
> 
> 
> Ready for review!

Thanks!

I just tested it and works fine for me!

You have changed services templates on your commit but it's not mentioned on your commit message or in PR description, please revert it (Keep one change per PR, if you like to make changes to service then do a separate PR for it)
<img width="1121" height="523" alt="image" src="https://github.com/user-attachments/assets/3ad150f7-6be8-43fc-a383-7e87f45e999f" />

favourch committed:
revert: remove unintended service template changes

- Revert changes to templates/service-templates.json and templates/service-templates-latest.json
- These changes were accidentally included in the repository search dropdown PR
- Service template updates should be in a separate PR

favourch commented:
@ShadowArcanist I've updated reverted the changes committed in error to templates/service-templates.json and templates/service-templates-latest.json

ShadowArcanist commented:
> @ShadowArcanist I've updated reverted the changes committed in error to templates/service-templates.json and templates/service-templates-latest.json

This PR still changes the service templates tho:
<img width="1675" height="1280" alt="image" src="https://github.com/user-attachments/assets/a63cad1f-29ce-414e-a66d-a1e6b2fda4e2" />


Your PR is meant to only editing the "github-private-repository.blade" so other file changes shouldn't be there

peaklabs-dev committed:
Merge branch 'next' into feat/github-app-repo-search-dropdown

peaklabs-dev committed:
style: fix newlines
